PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -12,12 +12,20 @@ local StarterGui = game:GetService('StarterGui')
 local GuiService = game:GetService('GuiService')
 local ContextActionService = game:GetService('ContextActionService')
 
+local LocalPlayer = PlayersService.LocalPlayer
+local PlayerGui = nil
+if LocalPlayer then
+	PlayerGui = PlayersService.LocalPlayer:WaitForChild("PlayerGui")
+end
+
 local VRServiceExists, VRService = pcall(function() return game:GetService("VRService") end)
 if not VRServiceExists or not VRService then
 	--No VRService? Fall back to UserInputService, it is the original VRService after all
 	VRService = UserInputService
 end
 
+local PortraitMode = false
```

Comment:
That's definitely the way to do it, didn't do this originally because not sure if Kyle's changes would be in before mine.